applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -152,7 +139,9 @@ def __build_cmd_get(cores: Core, device: AudioDevice, build: BuildType,
     if os.name == 'nt':
         release_flag = release_flag.replace('\\', '/')
     if pristine:
-        build_cmd += " -p"
+        build_cmd += " --pristine=auto"
```

Comment:
If we manually select pristine, i believe this script should just do:
```suggestion
        build_cmd += " --pristine"
```